packages/worker/src/runtime-options.ts

```diff
@@ -32,6 +32,14 @@ export interface RuntimeOptions {
    */
   telemetryOptions?: TelemetryOptions;
 
+  /**
+   * Interval for worker heartbeats. `null` disables heartbeating.
```

Comment:
I'm not totally comfortable using `null` as a user-facing marker to disable heartbeating. We do that internally at the bridge layer, to work around limitations of Node/NAPI/Neon, but I don't think we should do that in public APIs.

Instead, I suggest making this `Duration | boolean`, so `false` disables heartbeating, `true` or `== null` (which includes `undefined`) defaults to 60s, and anything else is parsed as a Duration.